hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationSource.java

```diff
@@ -34,6 +34,8 @@
 import static org.mockito.Mockito.when;
 import static org.mockito.internal.verification.VerificationModeFactory.times;
 
+import com.google.common.collect.Lists;
```

Comment:
This should come from hbase-thirdparty e.g. org.apache.hbase.thirdparty.com.google.common.collect.Lists

Reply:
Never mind, this is branch-1. ignore this comment. I will resolve it.

Reply:
@apurtell I can find guava in thirdparty, thus every place in hbase uses `Lists` directly from com.google.